### ISSUE TITLE
Get compute results

### ISSUE THREAD

**Issue by mihaisc** [oceanprotocol/vscode-extension#23]
Automatically download results in project folder
Have some settings in the extension to define the folder.

**Comment by paulo-ocean** [comment]
is this done already @jamiehewitt15 ?

**Comment by jamiehewitt15** [comment]
> is this done already @jamiehewitt15 ? afaik it is here https://github.com/oceanprotocol/vscode-extension/pull/27 right?

@paulo-ocean yeah, pending approval. The user selects the folder where the results should be downloaded.
